src/App.js

```diff
@@ -482,12 +484,13 @@ export default class App extends Component {
     this.setState({ view, scannerState: false }, cb);
   };
   changeAlert = (alert, hide = true) => {
+    console.log('Alert!!');
     clearTimeout(this.alertTimeout);
     this.setState({ alert });
     if (alert && hide) {
       this.alertTimeout = setTimeout(() => {
         this.setState({ alert: null });
-      }, 2000);
+      }, 200000);
```

Comment:
why?

Reply:
So you could read what's happening there?
It can be closed, by simply tapping at it.